arch/arm/src/cmake/platform.cmake

```diff
@@ -83,6 +83,6 @@ if(CONFIG_LIBSUPCXX_TOOLCHAIN)
   nuttx_find_toolchain_lib(libsupc++.a)
 endif()
 
-if(CONFIG_SCHED_GCOV)
+if(CONFIG_COVERAGE_TOOLCHAIN)
   nuttx_find_toolchain_lib(libgcov.a)
```

Comment:
do you need different library for clang

Reply:
This CONFIG_COVERAGE_TOOLCHAIN ​​depends on gcc. When using clang, we don't need to link